packages/cli/src/commands/install/mergeSupportFileImports.js

```diff
@@ -0,0 +1,24 @@
+const { runCodemod } = require('../../utils/runCodemod.js')
+const path = require('path')
+
+/**
+ * Copied from: https://stackoverflow.com/questions/1960473/get-all-unique-values-in-a-javascript-array-remove-duplicates
+ */
+const onlyUnique = (value, index, self) => self.indexOf(value) === index
+
+module.exports.mergeSupportFileImports = (support, paths) => {
+    const importNames = support.filter(onlyUnique)
+
+    const codemod = path.join(
+        __dirname,
+        '..',
+        '..',
+        '..',
```

Comment:
Could we simplify these by using a `paths.PROJECT_ROOT` constant instead of duplicating this lookup across all install controllers?

Reply:
I used `CLI_TOOL_ROOT` and `PROJECT` is ambigious